clikit/src/main/java/org/adamalang/clikit/model/Common.java

```diff
@@ -0,0 +1,101 @@
+package org.adamalang.clikit.model;
+
+import org.adamalang.clikit.exceptions.XMLFormatException;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import java.util.PrimitiveIterator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Common {
+    /** A utility class to support the model converting from XML to Java with functions like camelize */
```

Comment:
move to prior the "public class" as that will associate the comment with the class in javadoc